code/tests/component/test_SpawnsWaves.gd

```diff
@@ -0,0 +1,15 @@
+extends "res://addons/gut/test.gd"
+
+var component_cls = preload('res://src/component/SpawnsWaves.tscn')
+
+var comp
+
+func setup():
+	comp = component_cls.instance()
+	add_child(comp)
+
+func teardown():
+	remove_child(comp)
+
+func test_init():
+	assert_true(comp != null)
```

Comment:
I don't understand the point of these `init` tests. You're not testing any actual logic from the constructor.

Reply:
It's asserting that this component actually exists. That's pretty much it. I suppose it could be removed...

Reply:
I wouldn't necessarily remove them, but I wouldn't add them in the future.